### DISCUSSION TITLE
Why do I loose my clipboard when opening vim ?

### DISCUSSION THREAD

**Discussion by macjl** [isontheline/pro.webssh.net#1232] [Q&A] [answered]
Hello,

Here is my problem. When I have something in my clipboard, and logged through WebSSH in my Linux server, I can paste it in the terminal without issue.
But if I open vim, the content of the clipboard is lost. I can’t paste it into the vim screen, or any other application on my iPad.

Is there a setting in vim or WebSSH that I’m missing ?

**Comment by isontheline** [comment]
Hello @macjl 👋

That's strange, just tested under iPad and I can paste any value from my pasteboard

**Comment by macjl** [comment]
Here is a video describing the problem. You will also note some difficulties to unhide keyboard with Stage Manager, but this is not the topic here 😅

https://github.com/user-attachments/assets/6d8ad093-1f3f-4cfc-a873-01a346e8ee19

**Comment by macjl** [comment]
I did some other tests : 
- Same problem on Linux, and OpenWRT
- Same problem with vi and nano
- Same problem with a session in sh or bash
- But if I change the terminal to VT100, or ASCII, no more problem !

**Comment by macjl** [comment]
And same behavior with MacOS as SSH server with zsh !

**Comment by isontheline** [comment]
What's your clipboard access strategy?
<img width="461" alt="image" src="https://github.com/user-attachments/assets/7a10b41f-da68-4ab3-ad0e-3d2e4f4654ce">

**Reply by macjl** [reply]
Here it is : 
![IMG_0573](https://github.com/user-attachments/assets/24bf3a26-ca16-4ea5-b6dc-7616e4300424)

**Reply by isontheline** [reply]
When you put it on "ask", does it solve your issue?

**Reply by macjl** [reply]
Just tested in parallel, and nothing change. 😥

**Reply by macjl** [reply, marked as answer]
I just found the parameter that cause the issue. It’s here : 
![IMG_0575](https://github.com/user-attachments/assets/4e26fbad-9ae7-4c56-abdb-d22cab262d91)
 
 When set to « Copy to clipboard » the issue appears.

**Reply by isontheline** [reply]
Awesome! Thank you I have forgiven this one 😅
